### PR TITLE
added-profileBanner-feature

### PR DESCRIPTION
**Pull request by ozanbayram**
I added a new feature. it gets a profile banner photo. This is my first contributing I have done so far to open source projects.

### CODE REVIEW

File: twitter_scraper/modules/profile.py [hunk 1]
```diff
@@ -57,6 +58,8 @@ def __parse_profile(self, page):
 
         self.profile_photo = html.find(".ProfileAvatar-image")[0].attrs["src"]
 
+        self.banner_photo = html.find(".ProfileCanopy-headerBg img")[0].attrs["src"]
```
Comment (bisguzar) [inline twitter_scraper/modules/profile.py:61]: It will raise KeyError if banner photo not exists

Comment (ozanbayram) [reply twitter_scraper/modules/profile.py]: I fixed. Is it okey?

**Comment by bisguzar** [comment]
Thanks for fix, but you still need to update documentation from readme.md. Looks like you only updated example code, I can merge it if you update documentation.

**Comment by ozanbayram** [comment]
i don't understand what exactly should i update in readme.md

**Comment by ozanbayram** [comment]
I will add myself to contributors tomorrow.

**Comment by bisguzar** [comment]
To the table, 62-73. lines. I can add it by myself easily but you told its your first contribution. So its better if you do.

**Comment by ozanbayram** [comment]
The table is just about tweets with get_tweets(). But feature that I added is about profile with classProfile.  If you still want me to add, I will add 'banner_photo' to table.¹

**Comment by bisguzar** [comment]
You are right, my bad. I don't know why we don't have a attributes table for Profile class. Maybe I should crate it, thanks again for your contribution^^

**Pull request merged**